### PR TITLE
"New meaningoflife Cheat Sheet"

### PR DESCRIPTION
**Pull request by fozy81**
https://duck.co/ia/view/meaning_of_life

### CODE REVIEW

**Comment by daxtheduck** [comment]
@fozy81 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/meaningoflife.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "id": "meaningoflife_cheat_sheet",
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/json/meaningoflife.json]: @fozy81 This should be the same as the IA Page id (the one in the URL)

**Comment by MariagraziaAlastra** [comment]
Hey @fozy81, thanks for you contribution!
I just left a little comment in the code :smile:

**Comment by GuiltyDolphin** [comment]
@fozy81 Hey, thanks for contributing!

Unfortunately we already have a full Goodie that deals with this, its being [worked on](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2112) at the moment.

Perhaps you could do a cheat sheet for other Easter eggs?

**Comment by fozy81** [comment]
@GuiltyDolphin thanks, I'll work on average wing speed of an unladen swallow...but this was a good learning experience.

**Comment by moollaza** [comment]
> I'll work on average wing speed of an unladen swallow...but this was a good learning experience.

@fozy81 I think we may have that, but if not I would suggest you create a Goodie that works like some of the other Easter Eggs: https://github.com/duckduckgo/zeroclickinfo-goodies/blob/f9117a536f6305450dcdf8ba4d703e5b993a2c92/lib/DDG/Goodie/ValarMorghulis.pm

**Comment by moollaza** [comment]
I'll close this for now :+1:

**Comment by moollaza** [comment]
@fozy81 thanks again for trying to create this Meaning of Life Cheat Sheet! You’ve did a good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Pull request closed without merging**